src/main/java/frc/robot/Constants.java

```diff
@@ -13,18 +15,32 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-  public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
-  }
-
-    public static final CANAddress SWERVE_TURN_LEFT_FRONT = new CANAddress(0);
-    public static final CANAddress SWERVE_TURN_LEFT_REAR = new CANAddress(0);
-    public static final CANAddress SWERVE_TURN_RIGHT_FRONT = new CANAddress(0);
-    public static final CANAddress SWERVE_TURN_RIGHT_REAR = new CANAddress(0);
-    public static final CANAddress SWERVE_DRIVE_LEFT_FRONT = new CANAddress(0);
-    public static final CANAddress SWERVE_DRIVE_LEFT_REAR = new CANAddress(0);
-    public static final CANAddress SWERVE_DRIVE_RIGHT_FRONT = new CANAddress(0);
-    public static final CANAddress SWERVE_DRIVE_RIGHT_REAR = new CANAddress(0);
+    public static class OperatorConstants {
+        public static final int kDriverControllerPort = 0;
+    }
```

Comment:
This is redundant with the `XBOXCONTROLLER`  constant lower down in this file. Choose which of them you want to use and remove the other.